//comit
#include <QThread>

#include <QtConcurrent/QtConcurrent>
#include <QFuture>
#include <QMainWindow>
#include "ui_firstWindow.h"

class firstWindow;

class firstWindow;

class firstWindow;

class firstWindow;

class firstWindow;

class firstWindow;

class firstWindow : public QMainWindow
{
Q_OBJECT

public:
    explicit firstWindow(QWidget *parent = Q_NULLPTR);
    ~firstWindow();

    void DisplayGrid();

private:
    Ui::mainWindow *ui;

private slots:
    void on_Start_clicked();

};



